### ISSUE TITLE
Include prompt_id into langchain models

### ISSUE THREAD

**Issue by hamedizadpanah-ibm** [IBM/ibm-generative-ai-node-sdk#26]
Thank you for including prompt_id in the generate function, which has resolved 50% of my issues!

Can you please also include this in the langchain GenAIModel class?

i.e. instead of 
```typescript
import { GenAIModel } from '@ibm-generative-ai/node-sdk/langchain';

const model = new GenAIModel({
  modelId: 'bigscience/bloom',
  stream: true,
  parameters: {},
  configuration: {
    apiKey: 'pak-.....',
  },
});
``` 

so we can use 
```typescript
import { GenAIModel } from '@ibm-generative-ai/node-sdk/langchain';

const model = new GenAIModel({
  prompt_id: 'abc123',
  stream: true,
  configuration: {
    apiKey: 'pak-.....',
  },
});
```
https://github.com/IBM/ibm-generative-ai-node-sdk/blob/main/src/langchain/llm.ts

**Comment by Tomas2D** [comment]
Sure, I will personally include it and release it tomorrow.

**Comment by Tomas2D** [comment]
Released in 1.2.1
